src/pudl/metadata/fields.py

```diff
@@ -466,6 +466,18 @@
         "type": "number",
         "description": "Fraction of potential generation that was actually reported for a plant part.",
     },
+    "capacity_factor_offshore_wind": {
+        "type": "number",
+        "description": "Fraction of potential generation calculated for offshore wind.",
+    },
+    "capacity_factor_onshore_wind": {
+        "type": "number",
+        "description": "Fraction of potential generation calculated for onshore wind.",
```

Comment:
Do you want to specify about the 100m here?

Reply:
good point!